doc/man/man3/crypto_key_exchange.3monocypher

```diff
@@ -43,13 +43,11 @@ Do not use the same private key for both key exchanges and signatures.
 The public keys are different, and revealing both may leak information.
 .It Fa their_public_key
 The public key of the other party.
-.It Fa public_key
-The public key, generated from the
+.It Fa your_public_key
+Your public key, generated from
 .Fa secret_key
 with
 .Fn crypto_x25519_public_key .
-.It Fa secret_key
-A 32-byte random number, known only to you.
```

Comment:
I was surprised to see secret key appear twice!

If you agree maybe monocypher.h could be changed to match the docs too